src/avfoundation.cs

```diff
@@ -9145,13 +9168,27 @@ interface AVCapturePhotoSettings : NSCopying
 		[Export ("availableEmbeddedThumbnailPhotoCodecTypes")]
 		NSString[] _GetAvailableEmbeddedThumbnailPhotoCodecTypes { get; }
 
+		[Obsoleted (PlatformName.iOS, 12, 0, message: "Use AvailableEmbeddedThumbnailPhotoCodecTypes instead.")]
```

Comment:
that's us (not Apple) deprecating the API right ?
if so use `[Obsolete ("...")]` and put the API under `#if !XAMCORE_4_0` so it will be removed in a future profile without any additional work on our part

Reply:
and that means the new, correct API should be `virtual` (it's not below), so you need to reverse your `[Wrap]` ( both for `virtual` and to allow it to compile once `XAMCORE_4_0` is defined).

Reply:
Also please use '' for 'AvailableEmbeddedThumbnailPhotoCodecTypes'.